tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -322,6 +324,8 @@ def test_count_transaction_with_valid_condition(self):
         )
 
     def test_count_transaction_with_invalid_condition(self):
+        now = timezone.now()
```

Comment:
My idea at first was to not use `now` at all because the test didn't need any time freezing due to the failure happening before query execution, but for uniformity I agree we can use `@freeze_time` in both.

Reply:
so please do :)